applications/osb-portal/src/components/common/OSBDialog.tsx

```diff
@@ -18,14 +18,34 @@ import "react-markdown-editor-lite/lib/index.css";
 
 import * as Icons from "../icons";
 
+import {
+  checkBoxColor,
+} from "../../theme";
+
 interface DialogProps {
   open: boolean;
-  title: string;
+  title: string | React.ReactNode;
 
   actions?: React.ReactElement;
   closeAction: () => void;
 }
 
+const useStyles = makeStyles((theme) => ({
+  dialog: {
+    padding: 0,
+    "& .MuiDialogContent-root": {
+      padding: 0,
+      backgroundColor: '#3b3b3b',
```

Comment:
Better change to the closest color in the theme variables.

Reply:
Ok